### ISSUE TITLE
Support display of images

### ISSUE THREAD

**Issue by robertlugg** [umlet/umlet#401]
I would like to have an element that can display a bitmap (from file).  Perhaps this is possible with a custom element?  My specific usage is that I want to put a screenshot of a GUI in one of my objects.

**Comment by afdia** [comment]
This is currently not possible, but perhaps we will add the feature in the future (probably embedding the image within the uxf as base64 encoded string)

**Comment by robertlugg** [comment]
Yes, I'm sure that scaling will be a real pain.  But it would be useful.
